dev-workflow-ui/webContent/view/home.xhtml

```diff
@@ -48,10 +48,15 @@
                   </div>
                 </p:column>
               </p:dataTable>
-              <h:panelGrid rendered="#{empty homepageIvyDevWfBean.lastStarts}">
-                <h:outputText>You haven't started any processes yet</h:outputText>
-                <p:button value="Go to starts page" outcome="starts.xhtml" icon="si si-navigation-menu" />
-              </h:panelGrid>
+              <h:panelGroup rendered="#{empty homepageIvyDevWfBean.lastStarts}" layout="block"
+                styleClass="flex flex-column h-full">
+                <div>
+                  You haven't started any processes yet, <p:link value="go to process list" outcome="starts.xhtml"/>.
+                </div>
+                <div class="flex justify-content-center align-items-center h-full" style="min-height: 8rem;">
+                  <i class="si si-folder-empty text-7xl" />
```

Comment:
I would use the process icon here. I think it is called hierarchy or something (with rotate 90)